examples/scene/scene.rs

```diff
@@ -1,6 +1,7 @@
+//! This example illustrates loading and saving scenes from files
```

Comment:
This could really use a brief explanation of what a "scene" is in Bevy.

Reply:
For me to add that, I'd need to understand that myself 😅. From what I understand, a `Scene` is a newtype wrapper around a `World` so we can put complete worlds - for example loaded from gltf assets - into our main world. That works by copying all components from the scene to the world its spawned in when the command is executed. Therefor,  it does not really 'mount' other worlds into our app.world, it just uses the container to spawn a coupled set of components/entities into it?

So 'a `Scene` is a set of entites that can be loaded from an asset and added to our world'?.

Reply:
Yep, we can clean this up later :)